.github/workflows/backport-pull-request.yml

```diff
@@ -33,6 +33,10 @@ jobs:
         )
       )
     steps:
+      - run: |
+          echo '{
+            "prTitle": "{commitMessages} [backport to {targetBranch}]"
```

Comment:
The linked example from the test plan has the suffix _backport backport-v1.8_ which doesn't match this pattern. Was that one created with an older version?

Reply:
Can we assume that `commitMessages` will always be just one message ("merge PR ...")?

Reply:
> The linked example from the test plan has the suffix _backport backport-v1.8_ which doesn't match this pattern. Was that one created with an older version?

Yes. Noticed the missing to after testing.

Reply:
> Can we assume that `commitMessages` will always be just one message ("merge PR ...")?

- I cannot answer this, as I have no access to the respective repository settings.
- If the repository only allows squash merge commits. The answer is "yes".
- From my experience the repository setting is to only allow "squash merge".